component_documentation/componentDoc.css

```diff
@@ -163,6 +167,38 @@ body {
     .menu-hamburger {
         display: initial;
     }
-   
+}
+
+/* *** main container *** */
+
+.component-container {
+    padding: 1rem;
+    width: 90%;
+    box-sizing: border-box;
+    border: 1px solid var(--secondary-color);
+    margin: auto;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+}
+
+.sub-component-container {
+    box-sizing: border-box;
+    margin: 1rem;
+    padding: 1rem;
+    margin-bottom: 2rem;
+}
+
```

Comment:
don't repeat border-box property.you can use body{box-sizing:border-box} in main css.

Reply:
thank you. Ok